### PR TITLE
fix bugs and update UI

### PR DESCRIPTION
**Pull request by duy-nguyentuong**

### CODE REVIEW

**Comment by duy-nguyentuong** [comment]
Screen Shot 2020-08-07 at 9 29 00 PM
Screen Shot 2020-08-07 at 9 29 54 PM
Screen Shot 2020-08-07 at 9 30 18 PM
transaction

File: ts/prime/Api/src/util/string.ts [hunk 1]
```diff
@@ -9,3 +9,18 @@ export const camelize = (str: string): string => {
     })
     .replace(/\s+/g, '')
 }
+
+export const getAccountNumber = (number: string): string => {
```
Comment (markusait) [inline ts/prime/Api/src/util/string.ts:13]: I think you can increase readability a bit more here

```typescript
export const getCensoredAccountNumber = (number: string): string => {
  	const hideString = '*'
  	const lastCount = 4
	return hideString.repeat(number.length-lastCount) + number.slice(-lastCount)
}
```

File: ts/prime/Web/src/components/kyc/businessaddressdetails.tsx [hunk 2]
```diff
@@ -122,11 +122,14 @@ const BusinessAddressDetails: React.FC<BusinessAddressDetailsType> = ({
...
-            value={values.businessAddress?.address.country}
-            className="input-group"
-            onChange={(value: string, _) => {
-              handleChangeByName('businessAddress.address.country', value.toUpperCase())
+            className="text-input"
+            onChange={(countryName, option) => {
+              const countryCode = option['key']
+              handleChangeByName('tax.country', countryCode)
+              handleChangeByName('address.country', countryCode)
+              handleChangeByName('address.country_name', countryName)
```
Comment (markusait) [inline ts/prime/Web/src/components/kyc/businessaddressdetails.tsx:130]: Please sync with @masterhung0112 and see if he has a different solution for the country dropdown component

**Comment by markusait** [comment]
Added loader and success page to USDS withdrawals

File: ts/prime/Web/src/container/transfer/TransferWireWithdrawal.tsx [hunk 3]
```diff
@@ -38,12 +44,12 @@ const TransferWireWithdrawalContainer: React.FC<ContainerRouterProps> = (props:
       const bankAccounts = val.bankAccounts.map((bankAccount) => {
         let accountNum = ''
         if (bankAccount.last4) {
-          accountNum = 'xxxx xxx ' + bankAccount.last4
+          accountNum = 'xxxx xxxx ' + bankAccount.last4
```
Comment (Git-on-my-level) [inline ts/prime/Web/src/container/transfer/TransferWireWithdrawal.tsx:47]: why still use add the prefix when we have that logic in `getCensoredAccountNumber`

Comment (duy-nguyentuong) [reply ts/prime/Web/src/container/transfer/TransferWireWithdrawal.tsx:47]: Currently we only store the last 4 numbers into transfer targets

**Pull request merged**